### ISSUE TITLE
Supported Devices

### ISSUE THREAD

**Issue by tonydspaniard** [mopius/ndef-nfc#5]
I would like to ask what devices this library supports. I am currently searching for a solution to create a node app using node-webkit that reads/writes Tags using a http://www.acs.com.hk/en/products/3/acr122u-usb-nfc-reader reader. 

Found an option by creating a Chrome App (https://www.npmjs.com/package/chrome-nfc) but I would rather create an app that doesn't require another app (Chrome) in order to be launch or used. 

Any information would be highly appreciated.

**Comment by mopius** [comment]
The library is essentially formatting the NDEF standard by the NFC Forum that is in use in most NFC tags and also in device-to-device communication. It can parse NDEF-based NFC tag contents, as well as create those contents.

As such, the library is not directly interfacing with the NFC hardware of a device - for this, you need to use the NFC drivers / APIs that your platform provides (e.g., the Proximity APIs of Windows).

In short: you use the platform APIs to read / write the raw NFC information, and the NDEF library to make sense of these bits & bytes.
